Makefile

```diff
@@ -109,15 +109,15 @@ clean: ## Clean build artifacts
 clean-all: clean ## Clean all artifacts
 	rm -rf ${DIST_DIR} web/dist docs/build docs/node_modules web/node_modules
 	# delete generated
-	rm -rf docs/docs/40-cli.md docs/swagger.json
+	rm -rf docs/docs/40-cli.md docs/openapi.json
 
 .PHONY: generate
 generate: install-tools generate-swagger ## Run all code generations
 	CGO_ENABLED=0 go generate ./...
 
-generate-swagger: install-tools ## Run swagger code generation
-	swag init -g server/api/ -g cmd/server/swagger.go --outputTypes go -output cmd/server/docs
-	CGO_ENABLED=0 go generate cmd/server/swagger.go
+generate-swagger: install-tools ## Run openapi code generation and format it
```

Comment:
if we rename it all ... I think we can rename the make targets too ...

Reply:
renamed it